app/components/partials/Navbar.js

```diff
@@ -1,20 +1,35 @@
 import React from 'react';
 import { Link } from 'react-router';
 
+const iconClose = 'https://cdnjs.cloudflare.com/ajax/libs/material-design-icons/3.0.1/navigation/1x_web/ic_close_white_48dp.png';
+const iconMenu = 'https://cdnjs.cloudflare.com/ajax/libs/material-design-icons/3.0.1/navigation/1x_web/ic_menu_white_48dp.png';
+
+function toggleDropdown() {
+  var img = document.getElementById('dropdown-trigger');
+  img.src = (img.src == iconClose ? iconMenu : iconClose);
+  var x = document.getElementsByClassName('dropdown-content');
+  for (var i = 0; i < x.length; i++) {
+    x[i].classList.toggle('show');
+  }
```

Comment:
actually, this is because there used to be multiple elements with class `dropdown-content`. Now there's only one and by all means, we can even make it an id to make things easier, but this little extra code ensures that we can add more `dropdown-content`s in the future without a rewrite.